lib/sources/node.js

```diff
@@ -39,7 +40,13 @@ NodeSource.prototype.update = function(done) {
 };
 
 NodeSource.prototype._parse = function(body) {
-  var versions = _.unique(body.match(SEMVER));
+  try {
+    var inventory = toml.parse(body);
+  } catch (e) {
+    console.error("Parsing error on line " + e.line + ", column " + e.column + ": " + e.message);
```

Comment:
issue: in case of error here, you don't exit. Hence the method keeps executing even though `inventory` is empty. Shouldn't we have a fallback in case it fails here?

Reply:
Oh yes, sure :man_facepalming:, working on it :)